README.md

```diff
@@ -77,143 +72,160 @@ Among other things, you will find in this repository instructions on how to simu
+# Building ROS 1 creates conflicts with this ROS 2.0 workspace. Next line ensures the workspace is completely ROS 2.0.
 sed -i 's#/opt/ros/melodic#/opt/ros/crystal#g' ~/ros2_mara_ws/install/setup.bash
 ```
 
-#### Set up MoveITt! (for now in ROS)
+### Set up (optional) MoveIt! in ROS 1
 
-```bash
+```sh
 mkdir -p ~/ros_mara_ws/src
-cd ~/ros_mara_ws/src
+cd ~/catkin_mara_ws/src
 git clone https://github.com/AcutronicRobotics/MARA_ROS1
-cd ~/ros_mara_ws/
+cd ~/catkin_mara_ws/
 catkin_make_isolated --install
 ```
 
-#### Usage with Gazebo Simulation
-
-There are launch files available to bringup the MARA robot with MoveIt! and Rviz2, altogether.
-
-Don't forget to source the correct setup shell files and use a new terminal for each command!
+### Usage in Gazebo Simulator
 
-##### Terminal 1:
+#### Terminal 1 (ROS 2.0):
 
-To bring up the simulated robot in Gazebo. You can choose one of the following ros2 launch depends on the gripper that you want to use:
+To spawn the simulated robot in Gazebo, you can choose one of the following ros2 launch files depending on the gripper that you want to use:
 
 ```
 source ~/ros2_mara_ws/install/setup.bash
-ros2 launch mara_gazebo mara.launch.py
-ros2 launch mara_gazebo mara_gripper_140.launch.py
-ros2 launch mara_gazebo mara_gripper_85.launch.py
-ros2 launch mara_gazebo mara_gripper_hande.launch.py
 ```
 
-##### Rviz2
-
-To visualize the robot with Rviz2, you should type the following instructions:
 
 ```
-source ~/ros2_mara_ws/install/setup.bash
-rviz2
+ros2 launch mara_gazebo mara.launch.py
 ```
 
-##### MoveIT!
-
-###### ROS 2.0
-
-###### Terminal 1:
-
-You can run `gazebo`, spawn the model, publish the robot state and run `hros_cognition_mara_components` using the ROS 2.0 launch file. You can choose one of the following `ros2 launch` depends on the gripper that you want to use:
+**Optionally**, you can launch one of these other launch files, which correspond to other grippers.
 
 ```
-source ~/ros2_mara_ws/install/setup.bash
-ros2 launch mara_gazebo mara.launch.py
 ros2 launch mara_gazebo mara_gripper_140.launch.py
 ros2 launch mara_gazebo mara_gripper_85.launch.py
 ros2 launch mara_gazebo mara_gripper_hande.launch.py
 ```
 
-###### Terminal 2:
+#### Terminal 2 (ROS):
+```
+source ~/catkin_mara_ws/devel/setup.bash
+
+python3 ~/catkin_mara_ws/src/mara_camera/mara_bringup/scripts/follow_joints_trajectory_actions.py ~/catkin_mara_ws/src/mara_camera/mara_bringup/config/motors.yaml &
+# change the prefix to match with the gripper used in the Terminal 1
+roslaunch mara_bringup mara_bringup_moveit_actions.launch prefix:=140 &
 
-We need to run this node to create a bridge bewteen ROS and ROS 2.0. The topics that will be available are `/mara_controller/state`, `/joints_state` and `hros_actuation_servomotor_*********/trajectory`. Type the following command to run the bridge:
+sleep 4
+roslaunch mara_moveit_config mara_moveit_planning_execution.launch
```

Comment:
This line is launched inside `mara_bringup mara_bringup_moveit_actions.launch prefix:=140`

Reply:
Yep, this can be removed.